i18n-util/src/LocaleUtil.js

```diff
@@ -0,0 +1,43 @@
+const LOCALE_COOKIE_KEY = 'okta_user_lang';
+
+const getLocale = () => {
+  const supportedLocales = [
+    'en',
+    'ja',
+  ];
+  // check locale passed in query param
+  let locale = document.location.hash.split('?locale=')[1];
+  if (!locale) {
+    // check locale in cookie
+    if (document.cookie.indexOf(LOCALE_COOKIE_KEY) !== -1) {
+      locale = getCookieValue(LOCALE_COOKIE_KEY);
+    } else {
+      // check locale in browser
+      locale = navigator.language;
+    }
+  }
+
+  // default to en
+  if (!supportedLocales.includes(locale)) {
+    console.info('using default locale');
```

Comment:
I'd assume we want to strip these console statements in production?

Reply:
yes, will remove